pkg/cri/config/config.go

```diff
@@ -351,6 +351,13 @@ type PluginConfig struct {
 	// The string is in the golang duration format, see:
 	//   https://golang.org/pkg/time/#ParseDuration
 	ImagePullProgressTimeout string `toml:"image_pull_progress_timeout" json:"imagePullProgressTimeout"`
+	// DrainExecSyncIOTimeout is the maximum duration to wait for ExecSync
+	// API' IO EOF event after exec init process exits. A zero value means
+	// there is no timeout.
+	//
+	// The string is in the golang duration format, see:
+	//   https://golang.org/pkg/time/#ParseDuration
```

Comment:
for the config doc... suggest providing some examples otherwise to easy to fail 

"5h", "1m30s", "10s"

note that "" fails with invalid duration "" and "10" fails with: missing unit in duration "10"

Reply:
Updated. And the CRI plugin always validates the input, no matter what the user input, including empty value.
The CRI plugin will fail with any invalid input.